packages/cli/src/ui/components/IDEContextDetailDisplay.tsx

```diff
@@ -30,7 +32,8 @@ export function IDEContextDetailDisplay({
       paddingX={1}
     >
       <Text color={Colors.AccentCyan} bold>
-        IDE Context (ctrl+e to toggle)
+        {detectedIdeDisplay ? `${detectedIdeDisplay} ` : ''}IDE Context (ctrl+e
```

Comment:
IDE seems a bit repetitive here. We could probably get away with only showing the `detectedIdeDisplay` so it shows as `VSCode Context`